lib/runner.js

```diff
@@ -461,6 +461,36 @@ internals.executeTests = function (experiment, state, skip, callback) {
 };
 
 
+internals.experimentHasTests = function (experiment, state) {
+
+    if (experiment.experiments.length) {
```

Comment:
purely personal opinion but I find `> 0` more clear as to what you are doing, just a fyi